docs/MAINTAINERS.md

```diff
@@ -0,0 +1,54 @@
+# Maintainers
+
+Among our many awesome contributors, Snap has some great maintainers and plugin maintainers. Maintainers are expected to have a solid understanding of Snap and are here to help guide Snap, the plugins and the community forward in a positive direction.  
```

Comment:
No need to call out plugin maintainers as different than regular maintainers since we've revised the team model.